bungee/src/main/java/com/github/games647/fastlogin/bungee/FastLoginBungee.java

```diff
@@ -84,11 +85,17 @@ public void onDisable() {
     }
 
     private void registerHook() {
-        Plugin plugin = getProxy().getPluginManager().getPlugin("BungeeAuth");
-        if (plugin != null) {
+        Plugin plugin1 = getProxy().getPluginManager().getPlugin("BungeeAuth");
```

Comment:
Please give both plugin variables a more meaningful name. Otherwise mistakes could happen easily. Something like `bungeAuthPlugin` and `authenticatorPlugin`.